app/models/image.rb

```diff
@@ -16,6 +16,7 @@ class Image < ActiveRecord::Base
   validates :uid, presence: :true
   validates :photo, presence: :true, unless: :remote_url_provided?
   do_not_validate_attachment_file_type :photo_file_name
+  validates_attachment_content_type :photo, :content_type => ['image/jpeg', 'image/png', 'image/jpg']
```

Comment:
This looks great, but can we add `gif` too? Thanks!!

Reply:
@jywarren, sure I'll do.